direnvrc

```diff
@@ -1,5 +1,34 @@
 # shellcheck shell=bash
 
+# Usage: nix_direnv_version <version_at_least>
+#
+# Checks that the direnv version is at least old as <version_at_least>.
+nix_direnv_version() {
+  declare major=1 minor=2 patch=6
```

Comment:
Could you bump this version here as well? https://github.com/nix-community/nix-direnv/blob/adfca0d26dc687c1d10a4e88ef32cbc11bcaee47/scripts/create-release.sh#L19